packages/create-next-app/index.ts

```diff
@@ -43,6 +43,19 @@ const program = new Commander.Command(packageJson.name)
   .parse(process.argv)
 
 async function run(): Promise<void> {
+  // NodeJs Version Validation
+  var currentNodeVersion: Number = parseFloat(process.versions.node)
+
+  // Throws an Error if NodeJs Version is less than 10.13.x
+  if (currentNodeVersion < 10.13) {
+    console.error(
+      chalk.red(
+        `The version of NodeJs you are using is lower than the minimum required version (10.13.x)`
```

Comment:
`The current version ${currentNodeVersion} of NodeJs you are using is lower than the minimum required. (10.13)

This will be better because it will let the user know his current version